frontend/src/components/FeedListItem/FeedListItem.tsx

```diff
@@ -1,10 +1,63 @@
+import { useEffect, useState } from "react";
 import styles from "./FeedListItem.module.css";
 
 type FeedListItemProps = {
-  post: WorkoutTemplate,
+  post: WorkoutTemplate;
 };
 
 const FeedListItem: React.FC<FeedListItemProps> = ({ post }) => {
+  const [upvote, setUpvote] = useState<boolean | null>(null);
+  const [netVotes, setNetVotes] = useState<number>(0);
```

Comment:
I am wondering if you plan to use netVotes on client or not? I guess this will be used to rank the feed, then it might be more convenient if you calculate this and save it in backend, so that when you render the feed, you don't need to do this computation in real time.

Reply:
Yeah, I think that's a great idea to help me stop repeating code. I already moved the logic for updating the netVotes to the backend. So that partly addresses this. From what I'm understanding, you would want me to create another field in the workout Template model and update this value each time a vote action is made or are the changes I made in PR 29 fine.

Reply:
Actually, I think since this is just simple calculation and everything is already inside in the votes field, it's probably fine to not save it but calculate it when you need the ranking.